deploy/helm/values.yaml

```diff
@@ -0,0 +1,129 @@
+## Postgres params
+postgresql:
+  enabled: true
+  auth:
+    username: admin
+    password: admin
+    postgresPassword: admin
+    database: postgres
+  # Local dev purpose
+  # persistence:
+  #   existingClaim: postgresql-pv-claim
+  # volumePermissions:
+  #   enabled: true
+
+workflowService:
+  enabled: true
+  replicas: 1
+  strategyType: RollingUpdate
+  updateStrategy:
+    maxSurge: 1
+    maxUnavailable: "0"
+  nameOverride: workflowservice
+  migration: true
+  service:
+    port: 3000
+    type: ClusterIP
+    protocol: TCP  
+  image:
+    registry: index.docker.io
+    repository: gouthamappsmith/ball
```

Comment:
This one should stay?

Reply:
the registry, repository will change but the rest are required.